core/inflation/Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
-name        = "dc-inflation"
-description = "Darwinia economic inflation mechanism implementation."
-readme      = "README.md"
-authors     = { workspace = true }
-edition     = { workspace = true }
-version     = { workspace = true }
+authors.workspace = true
+description       = "Darwinia economic inflation mechanism implementation."
+edition.workspace = true
+name              = "dc-inflation"
+readme            = "README.md"
+version.workspace = true
```

Comment:
![image](https://user-images.githubusercontent.com/11801722/218631296-6b608bba-2e5e-4f32-95dd-9ead4e972cec.png)

Reported by the [Better TOML](https://marketplace.visualstudio.com/items?itemName=bungcip.better-toml) 

II am not sure which approach is the best practice.

Reply:
That extension is sucks.

Reply:
Why?

Reply:
```
Last updated | 2/18/2018, 4:50:44 PM
```

Reply:
Okay. I got it. Switch to another one.